File: AppWhere/app/src/main/java/com/example/appwhere/models/Sucursales.java
package com.example.appwhere.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Sucursales {

    @SerializedName("status")
    @Expose
    public Integer status;

    @SerializedName("description")
    @Expose
    public String description;

    @SerializedName("merchants")
    @Expose
    public List<Merchants> merchants = null;

    public Integer getStatus() {
        return status;
    }

    public void setStatus(Integer status) {
        this.status = status;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public List<Merchants> getMerchants() {
        return merchants;
    }

    public void setMerchants(List<Merchants> merchants) {
        this.merchants = merchants;
    }
}

File: AppWhereKotlin/app/src/main/java/com/example/appwherekotlin/models/Sucursales.kt
package com.example.appwherekotlin.models

import com.example.appwherekotlin.models.Merchants
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Sucursales {
    @SerializedName("status")
    @Expose
    var status: Int? = null

    @SerializedName("description")
    @Expose
    var description: String? = null

    @SerializedName("merchants")
    @Expose
    var merchants: List<Merchants>? = null

}